pkg/platform/kube/platform_test.go

```diff
@@ -1037,6 +1039,36 @@ func (suite *FunctionKubePlatformTestSuite) TestAlignIngressHostSubdomain() {
 	}
 }
 
+func (suite *FunctionKubePlatformTestSuite) TestEnrichFunctionWithUserNameLabel() {
+
+	functionName := "some-func"
+	functionConfig := *functionconfig.NewConfig()
+	authSession := auth.IguazioSession{
+		Username: "some-user",
+	}
+
+	// inject auth session to context
+	ctx := context.WithValue(suite.ctx, auth.AuthSessionContextKey, authSession)
+
+	createFunctionOptions := &platform.CreateFunctionOptions{
+		Logger:         suite.Logger,
+		FunctionConfig: functionConfig,
+		AuthSession:    &authSession,
+	}
+	createFunctionOptions.FunctionConfig.Meta.Name = functionName
+	createFunctionOptions.FunctionConfig.Meta.Labels = map[string]string{
+		"nuclio.io/project-name": platform.DefaultProjectName,
+	}
+
+	suite.Logger.DebugWith("Enriching function", "functionName", functionName)
+
+	err := suite.Platform.EnrichFunctionConfig(ctx, &createFunctionOptions.FunctionConfig)
+	suite.Require().NoError(err)
+
+	suite.Require().Empty(cmp.Diff(createFunctionOptions.FunctionConfig.Meta.Labels[iguazio.IguzioUsernameLabel],
```

Comment:
```suggestion
	suite.Require().Equal("some-user", createFunctionOptions.FunctionConfig.Meta.Labels[iguazio.IguzioUsernameLabel]),
```

`cmp.Diff` is used for complexed struct/type equality assertions. for premitives and less complexed, you can use the builtin assertion
`suite.Require().Equal(a, b)`

Reply:
Good to know 👍🏻